interface/ceed-basis.c

```diff
@@ -1059,6 +1146,18 @@ int CeedBasisGetGrad1D(CeedBasis basis, const CeedScalar **grad_1d) {
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Get divergence matrix of a CeedBasisHdiv
+  @param basis        CeedBasisHdiv
+  @param[out] div  Variable to store divergence matrix
+  @return An error code: 0 - success, otherwise - failure
+  @ref Advanced
+**/
+int CeedBasisGetDiv(CeedBasis basis, const CeedScalar **div) {
+  *div = basis->div;
```

Comment:
Should there be a check for when this is unsupported?

Reply:
We can add that switch case `basis_space` (with those magic numbers or whatever we will do) to work only for H(div).